auto_landing/include/auto_landing/eight_shape_traj.hpp

```diff
@@ -0,0 +1,32 @@
+#include<ros/ros.h>
+
+#include<geometry_msgs/PoseStamped.h>
```

Comment:
club all third-party headers (ones you haven't written) together. so remove those 2 empty lines.